modules/sdk-coin-atom/src/lib/index.ts

```diff
@@ -1,12 +1,10 @@
+import * as Constants from './constants';
 import * as Utils from './utils';
-import * as Interface from './iface';
 
+export {
+  CosmosTransaction as Transaction,
+  CosmosTransactionBuilder as TransactionBuilder,
+} from '@bitgo/abstract-cosmos';
 export { KeyPair } from './keyPair';
-export { Transaction } from './transaction';
```

Comment:
Removal of all these exports completely from here is a breaking change since /src/index.ts exposes these to end users.

Reply:
Added some of the exports back from abstract-cosmos & added a breaking change footer in the commit message for the ones still being removed completely